From e9c63a85a9e0f9f86f9632e9e127c38f568eac58 Mon Sep 17 00:00:00 2001
From: Rui Abreu Ferreira <raf-ep@gmx.com>
Date: Thu, 18 Aug 2011 00:29:18 +0100
Subject: [PATCH 02/11] Fix debug messages

---
 src/qt/if_qtcmdsrv.cpp | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/qt/if_qtcmdsrv.cpp b/src/qt/if_qtcmdsrv.cpp
index 3645a9f..f1b31d7 100644
--- a/src/qt/if_qtcmdsrv.cpp
+++ b/src/qt/if_qtcmdsrv.cpp
@@ -63,7 +63,7 @@ serverGetVimNames(void)
 int
 serverSendReply(char_u *name, char_u *reply) 
 {
-	qDebug() <<__func__ << name << reply;
+	qDebug() <<__func__ << (char*)name << (char*)reply;
 	return -1;
 }
 
@@ -94,7 +94,7 @@ serverGetReply(HWND server, int *expr_res, int remove, int wait)
 int
 serverSendToVim(char_u *name, char_u *cmd, char **result, void *ptarget, int asExpr, int silent)
 {
-	qDebug() << name << cmd;
+	qDebug() << (char*)name << (char*)cmd;
 
 	return -1;
 }
@@ -102,7 +102,7 @@ serverSendToVim(char_u *name, char_u *cmd, char **result, void *ptarget, int asE
 void
 serverSetName(char_u *name)
 {
-	qDebug() << __func__ << name;
+	qDebug() << __func__ << (char*)name;
 
 }
 
-- 
2.3.1

