From 2897d8872cce84c133ec2f01351ae8e3ea30d9ea Mon Sep 17 00:00:00 2001
From: Renee D'Netto <rdnetto@gmail.com>
Date: Tue, 28 Jul 2020 06:21:31 +1000
Subject: [PATCH 1/2] Replaced non-portable use of XScreen with GdkMonitor

---
 src/procman.cpp | 8 ++++++--
 1 file changed, 6 insertions(+), 2 deletions(-)

diff --git a/src/procman.cpp b/src/procman.cpp
index 09ddc38..01093dc 100644
--- a/src/procman.cpp
+++ b/src/procman.cpp
@@ -347,8 +347,12 @@ procman_data_new (GSettings *settings)
     g_free (color);
 
     /* Sanity checks */
-    swidth = WidthOfScreen (gdk_x11_screen_get_xscreen (gdk_screen_get_default ()));
-    sheight = HeightOfScreen (gdk_x11_screen_get_xscreen (gdk_screen_get_default ()));
+    GdkMonitor *monitor = gdk_display_get_monitor_at_window(gdk_display_get_default(), gtk_widget_get_window(pd->app));
+    GdkRectangle srect;
+    gdk_monitor_get_workarea(monitor, &srect);
+
+    swidth = srect.width;
+    sheight = srect.height;
     pd->config.width = CLAMP (pd->config.width, 50, swidth);
     pd->config.height = CLAMP (pd->config.height, 50, sheight);
     pd->config.update_interval = MAX (pd->config.update_interval, 1000);
-- 
2.25.3

